src/PCRE.NET.Native/pcrenet_match.c

```diff
@@ -108,18 +107,19 @@ PCRENET_EXPORT(void, match)(const pcrenet_match_input* input, pcrenet_match_resu
 
 PCRENET_EXPORT(void, dfa_match)(const pcrenet_dfa_match_input* input, pcrenet_match_result* result)
 {
-    const auto matchData = pcre2_match_data_create(input->max_results, nullptr);
-    const auto context = pcre2_match_context_create(nullptr);
+    pcre2_match_data* matchData = pcre2_match_data_create(input->max_results, NULL);
+    pcre2_match_context* context = pcre2_match_context_create(NULL);
     callout_data callout;
 
     if (input->callout)
     {
-        callout = { input->callout, input->callout_data };
+        callout.callout = input->callout;
+        callout.data = input->callout_data;
         pcre2_set_callout(context, &callout_handler, &callout);
     }
 
-    const auto workspaceSize = std::max(20u, input->workspace_size);
-    auto workspace = std::make_unique<int[]>(workspaceSize);
+    const int workspaceSize = MAX(20u, input->workspace_size);
```

Comment:
Could you replace this macro with a ternary operator, and remove the macro definition?

Reply:
Removed the macro definition.